rules.services.yml

```diff
@@ -34,8 +34,13 @@ services:
     class: Drupal\rules\EventSubscriber\RedirectEventSubscriber
     tags:
       - { name: 'event_subscriber' }
+  rules.ui_route_enhancer:
+    class: Drupal\rules\Routing\RulesUiRouteEnhancer
+    arguments: ['@plugin.manager.rules_ui']
+    tags:
+      - { name: route_enhancer }
   rules.ui_route_subscriber:
-    class: Drupal\rules\Routing\UiRouteSubscriber
+    class: Drupal\rules\Routing\RulesUiRouteSubscriber
```

Comment:
it's a bit more consistent with the other routing class (enhancer), the RulesUI term and eases finding the subscriber when doing class lockups. I stumbled over it multiple times.